sdk/src/main/java/com/mapbox/maps/MapView.kt

```diff
@@ -261,10 +264,10 @@ open class MapView : FrameLayout, MapPluginProviderDelegate, MapControllable {
   /**
    * Set new maximum FPS for map rendering.
    *
-   * @param maximumFps new maximum FPS value that must be > 0.
+   * @param fps new maximum FPS value that must be > 0 and less than [MAX_POSSIBLE_FPS].
```

Comment:
since `MAX_POSSIBLE_FPS` is internal, probably this is not visible to user?

Reply:
hard to say however imo it's not that important, general idea should be clean enough

Reply:
I mean this documentation uses an internal var, which means the generated doc will have no reference to this value, without source code, the `MAX_POSSIBLE_FPS` is meaningless.